src/app/users/UserHome.jsx

```diff
@@ -48,11 +47,17 @@ class UserHome extends React.Component {
   }
 
   render() {
-    const { userId } = this.state;
+    const { userId } = this.props.location.state;
```

Comment:
I'm holding the whole user object in state now, will refactor after I rebase, just giving you a heads up.